code/game/objects/effects/spiders.dm

```diff
@@ -42,7 +42,7 @@
 /obj/structure/spider/stickyweb/CanPass(atom/movable/mover, turf/target, height=0)
 	if(height == 0)
 		return TRUE
-	if(istype(mover, /mob/living/simple_animal/hostile/poison/giant_spider))
+	if(istype(mover, /mob/living/simple_animal/hostile/poison/giant_spider) || istype(mover, /mob/living/simple_animal/hostile/poison/terror_spider))

```

Comment:
if(istype(mover, /mob/living/simple_animal/hostile/poison/giant_spider) || isterrorspider(mover))

Reply:
https://travis-ci.org/ParadiseSS13/Paradise/jobs/656056300#L455

Reply:
https://github.com/ParadiseSS13/Paradise/blob/488b51bc1d9e9d5610aa10183153a3a9346589b7/code/modules/mob/mob_helpers.dm#L683

Should be defined and valid anywhere.

Reply:
I know it should. But as you can see in travis and the previous commit, it wasn't

Reply:
It is because isterrorspider is defined in code\modules\mob\mob_helpers.dm.
If you move the definition of 'isterrorspider' from that file to code\__DEFINES\mobs.dm, and place it as the next line under the '#define ishostile' line, then it does compile.
You'd need to test that this doesn't break anything, but in general its better to re-use the existing isX check than do an istype check.

Reply:
ditto
you may need to rebase to upstream/master to get that in your commit, keep in mind